docs/release/trg-9/trg-9-01.md

```diff
@@ -0,0 +1,76 @@
+
+## Objective
+
+The objective is to ensure consistency, usability, and adherence to design standards across all user interfaces within the business application.
+
+## Criteria (Basic)
+
+### Styleguide Compliance
+
+- All user interface elements must match the predefined styles. This includes fonts, colors, button styles, input fields, and layout grids.
+  - [Storybook](https://eclipse-tractusx.github.io/portal-shared-components)
+- Icons and imagery should be consistent (note that images and icons must be open source compliant and license files are needed)
+  - [Image, Graphics & Diagrams](https://github.com/eclipse-tractusx/portal-assets/blob/v2.0.0/docs/developer/Technical%20Documentation/Others/Images%2C%20Graphics%20and%20Diagrams.md)
+
+### Responsiveness and Compatibility
+
+- The application must be fully responsive and provide a seamless experience across different devices and screen sizes.
+- The application must be tested with multiple browsers - in case any browsers are not supported, the same must be documented.
+
+### Accessibility
+
+- The application must meet the minimum accessibility standards (e.g., WCAG 2.1).
+- Optional: All user interface components must be navigable and usable with keyboard-only controls.
+- Optional: Screen reader compatibility must be tested and ensured.
+
+### User Experience
+
+- The application's user flow must align with the user experience outlined in the styleguide.
+- Interactive elements must provide feedback to the user, such as hover states, active states, and disabled states, as per the styleguide.
+- The application must have a user help area which is linked to the current deployed application version.
+- The application must have a loading state or spinner component that indicates when data is being fetched or processed.
+- The application must have a mechanism in place to prevent the user interface from being triggered a second time by the user while running loading processes (POST Data). This is important to avoid any unintended actions or disruptions caused by multiple triggers. By implementing this safeguard, the application can ensure a smooth and reliable user experience during loading processes.
+- The application must have a notification or alert component that displays important messages or updates to the user (success as well as errors).
+
+### Typography and Readability
+
+- Text must adhere to the styleguide's typography standards, including font sizes, line heights, and font weights.
+- Text contrast must meet the minimum contrast ratios for readability.
+
+### Localization and Internationalization
+
+- The application/user interface must support multiple languages, allowing users to switch between different language options.
```

Comment:
I would specify two minimum languages : EN, DE

Reply:
thanks for the good feedback
the idea was to transfer the previous state of the styleguide without any new requirements, I suggest to open a update pull request to the TRG once this has been merged to provide the necessary awareness about the change to all product teams

Reply:
thats fine for me